README.md

```diff
@@ -39,15 +39,19 @@ However, the `attribution` field will appear after the quote, behind a `~`, if g
 
 If you do not know who, or cannot find the original attribution, please use `Unknown`, or `Various`. If many have said it, use `Apocryphal`. But, if you're having trouble, you can always ask others, who may know it themselves or know where to check.
 
-If citing both a person and a work, please do this in a format that is sensible, perhaps `person, work`, inside the `attribution` field --- with a URL (i.e. for Tweets, videos, or articles) in the `source` field. We tend to prefer this is included in the `attribution`, as `source` is used for HTTP URLs. If the year or edition is relevant, include this either parenthised `(3rd edition)`, or following it like, say, `Unix Epoch, 1900`. With years, we prefer and assume Common Era, such as `Julius Caesar (100 - 44 BCE)` or `de Finibus Bonorum et Malorum, 45 BCE`.
+If citing both a person and a work, please do this in a format that is sensible, perhaps `person, work`, inside the `attribution` field --- with a URL (i.e. for Tweets, videos, or articles) in the `source` field. We tend to prefer this is included in the `attribution`, so use `source` for HTTP/HTTPS URLs (where relevant). If the year or edition is relevant, include this either parenthised `(3rd edition)`, or following it like, say, `Unix Epoch, 1970`. With dating, we prefer and assume [Common Era](https://en.wikipedia.org/wiki/Common_Era), such as `Julius Caesar (100 - 44 BCE)` or `de Finibus Bonorum et Malorum, 45 BCE`.
 
-When citing something said by a character, use a simple rule of thumb: if you know who wrote it, cite as Author, Character, otherwise cite the character directly. For example, The Doctor or Jean-Luc Picard are cited as themselves, because various scriptwriters and actors are involved, whereas Macbeth is cited as William Shakespeare, Macbeth, as we're quoting his script. Other examples, such as Sherlock Holmes, may be context dependent, such as the books will be cited as Arthur Conan Doyle, Sherlock Holmes, whereas film or show adaptations are cited accordingly (Character, Film/Show, Year).
+When citing something said by a character, use a simple rule of thumb: if you know who wrote it, cite as `Author, Character`, otherwise cite the character directly. For example, `The Doctor` or `Jean-Luc Picard` are cited as themselves, because various scriptwriters and actors are involved, whereas Macbeth is cited as `William Shakespeare, Macbeth`, as we're quoting his script (the character should be listed, though in the case of Macbeth, it ould be redundant). Other examples, such as `Sherlock Holmes`, may be context dependent, such as the books will be cited as `Arthur Conan Doyle, Sherlock Holmes`, whereas film or show adaptations are cited accordingly (`Character, Film/Show, Year`). Shows may include the specific episode by name and perhaps number, where relevant, but further details (timestamps etc.) should not be included (but may be part of a `source` URL, such as timestamped links to YouTube).
```

Comment:
"it ould be redundant", typo.

Reply:
Well spotted, I'll correct that momentarily.

Reply:
Corrected